data-gen/src/test/java/io/smartcat/data/loader/BuildRunnerTest.java

```diff
@@ -152,4 +140,23 @@ public void should_allow_setting_completelly_different_fields_in_different_build
         Assert.assertEquals(3, numberOfUsersWithFirstNameAndNumberOfCards);
     }
 
+    @Test
```

Comment:
That way I would be testing that an exception is thrown in the test, but not where it is thrown. This way I wrap the exact method that I expect to throw the exception.